x-pack/plugins/security_solution/server/lib/detection_engine/rule_actions_legacy/logic/notifications/legacy_rules_notification_alert_type.ts

```diff
@@ -7,6 +7,7 @@
 
 import type { Logger } from '@kbn/core/server';
 import { parseScheduleDates } from '@kbn/securitysolution-io-ts-utils';
+import { DEFAULT_APP_CATEGORIES } from '@kbn/core-application-common';
 import {
```

Comment:
@JiaweiWu 

Detection engine code looks good, but I've noticed the following behaviour

I created maintenance window for stack rule only - but got callout notification that my security rules under maintenance, although actions are triggering




https://github.com/elastic/kibana/assets/92328789/cfd75b01-8e6f-41c5-9d6a-26b677649c37

we are using this component for displaying messages: [https://github.com/elastic/kibana/blob/32562aa53ff07ab151d77b9b7559c54a80abd633/packages/kbn-alerts-ui-shared/src/maintenance_window_callout/index.tsx](/packages/kbn-alerts-ui-shared/src/maintenance_window_callout/index.tsx)

Looks like it does not support showing message depends on a category

Reply:
Yea good catch, the callout isn't aware of the category selection. I will fix that.

Reply:
Fix: https://github.com/elastic/kibana/pull/166781/commits/b40f2e7af9f54186df8a72d9cb1b3b12649f1621